src/intro-view/intro-view.html

```diff
@@ -10,6 +10,10 @@
       :host {
         display: block;
       }
+      img {
+        max-width: 80%;
+        max-height: 80%;
```

Comment:
There isn't a specific reason. My main focus was making the alterations in the shared-styles.